scripts/ci/_utils.sh

```diff
@@ -777,7 +777,9 @@ function build_image_on_ci() {
     echo "Finding changed file names ${TRAVIS_BRANCH}...HEAD"
     echo
 
-    CHANGED_FILE_NAMES=$(git diff --name-only "${TRAVIS_BRANCH}...HEAD")
+    git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+    git fetch origin ${TRAVIS_BRANCH}
+    CHANGED_FILE_NAMES=$(git diff --name-only "remotes/origin/${TRAVIS_BRANCH}...HEAD")
```

Comment:
when PR is not against master, and the PR is not a branch in apache/airflow, travis clones master, fetches the HEAD of this PR and checks it out. Then, this shell script tries `git diff --name-only v1-10-test...HEAD`, which errors since `v1-10-test` has not been fetched. These changes are to have `diff` output in all normal cases, plus this case. See for instance these Travis logs: https://travis-ci.org/apache/airflow/jobs/610354673#L166-L214

Reply:
Nice! Thanks!